snorkel/labeling/model/label_model.py

```diff
@@ -366,9 +366,9 @@ def predict_proba(self, L: np.ndarray) -> np.ndarray:
 
         Example
         -------
-        >>> L = np.array([[0, 0, 0], [1, 1, 1], [1, -1, 1]])
+        >>> L = np.array([[0, 0, 0], [1, 1, 1], [1, 1, 1]])
```

Comment:
Probably a worse example without an abstain. In general, this L is just not super realistic.